github/CodeScanAlertInstance.py

```diff
@@ -115,7 +101,7 @@ def _useAttributes(self, attributes: Dict[str, Any]) -> None:
         if "commit_sha" in attributes:  # pragma no branch
             self._commit_sha = self._makeStringAttribute(attributes["commit_sha"])
         if "message" in attributes:  # pragma no branch
-            self._message = self._makeDictAttribute(attributes["message"])
+            self._message = self._makeDictAttribute(attributes["message"])  # type: ignore
```

Comment:
`Dict[str, Any]` can't cast to `TypedDict`

Reply:
Or we can just use `dict[str,Any]`, remove `Message(TypedDict)`

Reply:
Happy to remove the `Message(TypedDict)`.